src/crypto/common/Algorithm.h

```diff
@@ -69,19 +69,20 @@ class Algorithm
         AR2_WRKZ,      // "argon2/wrkz"      Argon2id (WRKZ)
         ASTROBWT_DERO, // "astrobwt"         AstroBWT (Dero)
         KAWPOW_RVN,    // "kawpow/rvn"       KawPow (RVN)
+        RX_DEFYX,      // "defyx"            DefyX (Scala).
         MAX
     };
 
     enum Family : int {
-        UNKNOWN,
-        CN,
-        CN_LITE,
-        CN_HEAVY,
-        CN_PICO,
-        RANDOM_X,
-        ARGON2,
-        ASTROBWT,
-        KAWPOW
+        UNKNOWN
+        ,CN
+        ,CN_LITE
+        ,CN_HEAVY
+        ,CN_PICO
+        ,RANDOM_X
+        ,ARGON2
+        ,ASTROBWT
+        ,KAWPOW
```

Comment:
Is this really a good idea? Will cause 100% merge conflict in future when pulling from xmrig master branch.